crates/biome_js_analyze/tests/specs/nursery/useTrimStartEnd/invalid.js

```diff
@@ -0,0 +1,14 @@
+foo.trimLeft()
+foo.trimRight()
+trimLeft.trimRight()
+foo.trimLeft.trimRight()
+"foo".trimLeft()
+foo
+	// comment
+	.trimRight /* comment */
+	/* comment */
+	()
+foo?.trimLeft()
+bar['trimLeft']()
+bar["trimLeft"]()
+bar[`trimLeft`]()
```

Comment:
There seems no logic in the code to treat these cases as invalid. I think it's fine to ignore them in this PR. But in that case, we should remove them from the test cases and add a TODO comment.

Alternatively, you can also implement the logic to handle these cases.

wdyt?

Reply:
@Sec-ant 
Thank you for the feedback! I will implement the logic tomorrow:)